apps/wallet/src/ui/app/pages/initialize/backup/index.tsx

```diff
@@ -54,55 +53,127 @@ const BackupPage = ({ mode = 'created' }: BackupPageProps) => {
+                    </div>
+
+                    {mode === 'created' ? (
+                        <>
+                            <div className="mb-1 mt-7.5">
+                                <Text
+                                    variant="caption"
+                                    color="steel-darker"
+                                    weight="bold"
+                                >
+                                    Recovery phrase
+                                </Text>
+                            </div>
+                            <div className="mb-3.5 mt-2 text-center">
+                                <Text
+                                    variant="p2"
+                                    color="steel-dark"
+                                    weight="normal"
+                                >
+                                    Your recovery phrase makes it easy to back
+                                    up and restore your account.
+                                </Text>
+                            </div>
+                            <Loading loading={loading}>
+                                {mnemonic ? (
+                                    <div className="text-steel-dark flex flex-col flex-nowrap gap-2 self-stretch font-semibold text-heading5 p-3.5 rounded-[15px] bg-white border border-solid border-[#EBECED] shadow-button leading-snug">
+                                        {mnemonic}
+                                    </div>
+                                ) : (
+                                    <Alert>{error}</Alert>
+                                )}
+                            </Loading>
+                            <div className="mt-3.75 mb-1 text-center">
+                                <Text
+                                    variant="caption"
+                                    color="steel-dark"
+                                    weight="semibold"
+                                >
+                                    Warning
+                                </Text>
+                            </div>
+                            <div className="mb-1 text-center">
+                                <Text
+                                    variant="p2"
+                                    color="steel-dark"
+                                    weight="normal"
+                                >
+                                    Never disclose your secret recovery phrase.
+                                    Anyone with the passphrase can take over
+                                    your account forever.
+                                </Text>
+                            </div>
+                            <div className="flex-1" />
+                            <div className="w-full text-left flex mt-5">
+                                <label className="flex items-center justify-center h-5 mb-0 mr-5 text-sui-dark gap-1.25 relative cursor-pointer">
```

Comment:
If we could make a checkbox component would be ideal! Probably it's easy just to extract it to it's one file.

Reply:
Agreed,  I will create a separate task for this.